src/config/url.config.json

```diff
@@ -2,10 +2,12 @@
     "URLS": {
         "ORGANIZATION_CREATE_URL": "api/org/v1/create",
         "ORGNIZATION_SEARCH_URL": "api/org/v1/search",
-        "USER_BLOCK_URL": "api/user/v1/block",
         "USER_CREATE_URL": "api/user/v3/create",
         "USER_UPDATE_ROLE_URL": "api/user/v1/role/assign",
+        "USER_BLOCK_URL": "api/user/v1/block",
         "USER_UNBLOCK_URL": "api/user/v1/unblock",
+        "USER_DASHBOARD_ALL_USER_ROLE_URL":"api/data/v1/form/read",
+        "USER_DASHBOARD_USERS_ROLE_COUNT":"api/user/v1/search",
```

Comment:
use USER_SEARCH_URL instead of USER_DASHBOARD_USERS_ROLE_COUNT

Reply:
Done